pybossa/api/project.py

```diff
@@ -89,8 +90,15 @@ def _forbidden_attributes(self, data):
         for key in data.keys():
             if key in self.reserved_keys:
                 if key == 'published':
-                    raise Forbidden('You cannot publish a project via the API')
+                    raise Forbidden('You cannot publish a project via the API')   
                 raise BadRequest("Reserved keys in payload")
+    
+    def _forbidden_attributes_put(self, data):
+        for key in data.keys():
+            if key in self.put_reserved_keys:
+                if key == 'short_name':
+                    raise Forbidden('You cannot update short_name via the API')    
```

Comment:
Do we need the special handling? Or would `raise Forbidden('Cannot change {} via API'.format(key))` work?

Reply:
No special handling is needed its correct.  Deep suggested to not create a new function and instead use request.method in _forbidden_attributes() Im planning to do it that way.